jenkins/kernel-arch-complete.sh

```diff
@@ -224,6 +224,7 @@ if [[ BUILDS_FINISHED -eq 1 ]]; then
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "igt_drm_kms", "send_to": ["gtucker@collabora.com", "mgalka@collabora.com", "dan.rue@linaro.org", "matthew.hart@linaro.org", "broonie@kernel.org", "khilman@baylibre.com"], "format": ["txt"], "delay": 2700}' ${API}/send
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "cros-ec", "send_to": ["gtucker@collabora.com", "mgalka@collabora.com", "dan.rue@linaro.org", "matthew.hart@linaro.org", "broonie@kernel.org", "khilman@baylibre.com"], "format": ["txt"], "delay": 2700}' ${API}/send
         curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "sleep", "send_to": ["gtucker@collabora.com", "mgalka@collabora.com", "dan.rue@linaro.org", "matthew.hart@linaro.org", "broonie@kernel.org", "khilman@baylibre.com"], "format": ["txt"], "delay": 2700}' ${API}/send
+        curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'",  "report_type": "test", "plan": "sleep", "test_template": "plans/v4l2-compliance", "send_to": ["gtucker@collabora.com", "mgalka@collabora.com", "dan.rue@linaro.org", "matthew.hart@linaro.org", "broonie@kernel.org", "khilman@baylibre.com"], "format": ["txt"], "delay": 2700}' ${API}/send
```

Comment:
This line should be replacing the v4l2-compliance ones rather than add a new one.

(Also it's trying to send a "sleep" report with a v4l2 template)

Reply:
I just replaced one line not both to don't broke the tow of them if things are wrong